test-lib-openshift.sh

```diff
@@ -177,6 +188,18 @@ function ct_os_check_pod_readiness() {
 function ct_os_wait_pod_ready() {
   local pod_prefix="${1}" ; shift
   local timeout="${1}" ; shift
+  # If there is a build pod - wait for it to finish first
+  sleep 3
+  if ct_os_get_all_pods_name | grep -E "${pod_prefix}.*-build"; then
+    SECONDS=0
+    echo -n "Waiting for ${pod_prefix} build pod to finish ..."
+    while ! [ "$(ct_os_get_build_pod_status "${pod_prefix}")" == "Completed" ] ; do
+      echo -n "."
+      [ "${SECONDS}" -gt "${timeout}0" ] && echo " FAIL" && return 1
```

Comment:
```suggestion
      [ "${SECONDS}" -gt "${timeout}0" ] && echo " FAIL" && ct_os_get_status && return 1
```

Reply:
We already have: https://github.com/sclorg/container-common-scripts/blob/master/test-lib-openshift.sh#L60 for debugging output in case of a failure. Would rather have it in one place than in every function.